plugins/linux-capture/xcomposite-input.c

```diff
@@ -712,6 +714,11 @@ static void *xcompcap_create(obs_data_t *settings, obs_source_t *source)
 	s->show_cursor = true;
 	s->strict_binding = true;
 	s->source = source;
+	enum obs_nix_platform_type platform = obs_get_nix_platform();
+	s->egl = platform == OBS_NIX_PLATFORM_X11_EGL;
+	if (s->egl) {
+		s->strict_binding = false;
+	}
```

Comment:
Nitpick: can you remove the curly braces? Does clang-formatter complain about that?

Reply:
With or without is will not complain.